### PR TITLE
2023 constitution update, role changes, misc. small updates and fixes

### PR DESCRIPTION
**Pull request by EthanDenny**
# Rundown of Changes

### BIG THINGS

1. Replace the role of Social Representative with two new roles: Outreach Coordinator and Marketing Officer.
2. Replace Representative-At-Large and First Year Representative with a new, non-voting role: Society Volunteer
3. Remove the role of Hack Frost Rep (Madison, head of Hack Frost, has suggested that it is no longer necessary).
4. Add the role of Executive Alumni, a non-voting role for former members of the exec who wish to remain in an advisory capacity. We already have this unofficially (Honourary Executive).
5. Allow any member of the society to be appointed to an empty role, with preference given to Society Volunteers

### SMALL THINGS

1. Add a definition of a “Voting Role”: any role except Society Volunteer and Executive Alumni would be allowed to vote in meetings.
2. Remove the role of “Representative Advisor”.
3. Add a rule that if there is only one nominee (incumbent or incoming) for a position, then that nominee shall be acclaimed for that position.
4. Make an addition to the duties of the Science Society Representative, indicating that they should communicate any items which should be on the agenda at an exec meeting before the meeting so that they can be discussed.
5. Make an addition to the duties of the Tech Officer, indicating that they are responsible for managing the Discord server, Discord bots, and storing passwords and records relating to their duties.
6. Change the notice period for an exec meeting to 72 hours (rather than 24), or 24 in case of emergency (previously unspecified)
7. Change the rules for calling meetings to better specify which months meetings are necessary
8. Add a restriction for signing authorities that they may not have a “conflict of interest”

---

# APPROVALS

- [x] Ethan Denny (egdenny[at]mun.ca)
- [x] Emily Dormody (emdormody[at]mun.ca)
- [x] Amélie (amelierb[at]mun.ca)
- [x] Gizem Ensari (gensari[at]mun.ca)
- [x] Pratham (prathamn[at]mun.ca)
- [x] Leah Murphy (lmurphy19[at]mun.ca)
- [x] Parsa Esmkhani (pesmkhani[at]mun.ca)
- [x] Jack Harrhy (jaharrhy[at]mun.ca)
- [x] Ripudaman Singh (ripudamans[at]mun.ca)
- [x] Kanika Mathur (kmathur[at]mun.ca)

### CODE REVIEW

**Review by Kanika2003 — APPROVED** [review]

**Review by jackharrhy — APPROVED** [review]

File: src/pages/society/constitution.md [hunk 1]
```diff
@@ -266,8 +257,8 @@ In the event that a position is vacant either:
 ## 10.1. Date and Time
 
 - Only the President or Vice President may call an executive meeting;
-- Notice of time and place of each meeting of the Executive shall be sent to each executive, not less than twenty-four hours before the time of the meeting;
-- There shall be at least one Executive meeting per month during the academic year, excluding the months of April, May, June, July, August, and December;
+- Notice of time and place of each meeting of the Executive shall be sent to each executive, not less than seventy-two before the time of the meeting, or twenty-four hours in case of an emergency;
```
Comment (zcvaters) [inline src/pages/society/constitution.md:260]: Missing "hours" after "seventy-two"

**Review by zcvaters — APPROVED** [review]

File: src/pages/society/constitution.md (hunk 1 above)
Comment (EthanDenny) [reply src/pages/society/constitution.md]: Fixed

**Review by emilydormody — APPROVED** [review]

**Review by PrathamAtMun — APPROVED** [review]
lgtm

**Review by singhripudaman — APPROVED** [review]

**Comment by leahmarg** [comment]
ya

**Pull request merged**